### ISSUE TITLE
Add VB6 highlighting (.ctl files) based on the VBA syntax

### ISSUE THREAD

**Issue by MartinHickling101** [serkonda7/vscode-vba#74]
### Describe the feature

.ctl file not resolving.

### Proposed Solution

add .ctl to the list of available files

**Comment by serkonda7** [comment]
Could you please provide a short explanation or link what ctl files are used for?

**Comment by DecimalTurn** [comment]
`.ctl` files are for VB6 not VBA, but since the syntax between the two is very similar, some people might want to use VBA syntax highlighting for VB6.

I recently had to figure out the exact syntax differences between VBA and VB6 for this [PR](https://github.com/github-linguist/linguist/pull/6355), and the only 2 differences I could find were these:

> 1. [VBA7 has new 64-bit features that were not implemented in VB6](https://learn.microsoft.com/en-us/office/vba/language/concepts/getting-started/64-bit-visual-basic-for-applications-overview)
> 2. VBA has some unique top module declarations
> 	a. [Option Private Module](https://learn.microsoft.com/en-us/office/vba/language/reference/user-interface-help/option-private-statement#:~:text=Option%20Private%20is,in%20Visual%20Basic.)
> 	b. [Option Compare Database](https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-basic-6/aa266181(v=vs.60)#:~:text=OptionCompareDatabase%20can%20only%20be%20used%20within%20Microsoft%20Access)

**Comment by serkonda7** [comment]
Thanks for your input and thoughts!

> Just need to press Ctrl+Shift+P, then Change Language Mode and select VBA.

Until this language binding is a lot more requested, it has to be changed manually like above.

**Comment by DecimalTurn** [comment]
FYI, once this [PR](https://github.com/github-linguist/linguist/pull/6894) is merged, we will be using the VBA grammar provided here for syntax highlighting of VB6 code on GitHub.

**Comment by serkonda7** [comment]
Thank you for this detailed and thoughtout comment!

### TLDR
- Change current source.vba grammar to source.vb6
- Adjust file extensions
- Create new source.vba that includes source.vb6
- Consider the differences as bugs that can be fixed later


### Details
After some consideration, I would propose a another approach:

We use the technically correct dependency structure like in Option A
```mermaid
%%{init: {'theme':'neutral'}}%%
graph TD
    A[source.wwb]
    B[source.vba]
    C[source.vb6]

    A -- includes --> B
    B -- includes --> C

```

**BUT** reduce the initial migration effort by changing the current VBA grammar to the VB6 grammar as is.
The few invalid highlights will be considered bugs and could be fixed as noticed / reported. 

Please let me know what you think.

**Comment by DecimalTurn** [comment]
That seems like a good plan.
Let me know if you'd like me to propose a PR for this or if you prefer to take care of it.

**Comment by serkonda7** [comment]
I will do the basics this weekend.

**Comment by serkonda7** [comment]
@MartinHickling101, @JaspaBass what dou you think of the solution proposed in #126?
